.github/workflows/main.yml

```diff
@@ -23,6 +23,9 @@ jobs:
       - name: Check out the code
         uses: actions/checkout@v2
 
+      - name: Check for linting errors
+        run: make lint
```

Comment:
We wanna make sure now we run the linter before continuing with the tests so we fail early.